pyproject.toml

```diff
@@ -90,3 +90,11 @@ text = "\n---\n\n[Full changelog](https://github.com/encode/httpx/blob/master/CH
 [[tool.hatch.metadata.hooks.fancy-pypi-readme.substitutions]]
 pattern = 'src="(docs/img/.*?)"'
 replacement = 'src="https://raw.githubusercontent.com/encode/httpx/master/\1"'
+
+[tool.ruff]
+select = ["E", "F", "I", "B", "PIE"]
+ignore = ["B904", "B028"]
+line-length = 120
```

Comment:
Should we comment above here with a pointer to a relevant part of the docs?

Reply:
I've added a reference to the docs

Reply:
Is it enough?

Reply:
'tis